airbyte-integrations/bases/base-typing-deduping-test/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/BaseTypingDedupingTest.java

```diff
@@ -107,6 +109,19 @@ public abstract class BaseTypingDedupingTest {
    */
   protected abstract List<JsonNode> dumpRawTableRecords(String streamNamespace, String streamName) throws Exception;
 
+  /**
+   * Utility method for tests to check if table exists
+   *
+   * @param streamNamespace
+   * @param streamName
+   * @return
+   * @throws Exception
+   */
+  protected boolean checkTableExists(String streamNamespace, String streamName) {
+    // Implementation is specific to destination's tests.
+    return true;
```

Comment:
why not leave this as an abstract method?

Reply:
Probably will be abstract.  went with empty for now to get through ITs passing.  Intention was to have the disableTD tests to check Final table isn't created or exists.